CMakeLists.txt

```diff
@@ -66,7 +66,11 @@ FetchContent_MakeAvailable(repo-common repo-core)
 #
 if(${TRITON_ENABLE_GPU})
   set(CUDA_USE_STATIC_CUDA_RUNTIME OFF)
+  set(CMAKE_FIND_DEBUG_MODE TRUE)
```

Comment:
Why was this needed here? Should we be using FIND_DEBUG_MODE everywhere we are using find_package as well?

Reply:
For debugging Windows build issue, not to be merged in.